manifests/profile/prometheus/exporter/mysql.pp

```diff
@@ -25,5 +25,12 @@
     content => template('nebula/profile/prometheus/exporter/mysql/defaults.sh.erb')
   }
 
+  @@concat_fragment { "prometheus mysql service ${::hostname}":
```

Comment:
This will need to be imported by the prometheus profile—right now, it's being exported without being imported anywhere.

Reply:
This still isn't being imported by prometheus—I'd look for `Concat_fragment <<|` and `concat_file` in the prometheus profile for examples to imitate. I think you'll also need to edit the prometheus config template file to tell it to read the resulting concat file.

Reply:
Sorted this out and made sure docker was mounting the file.